### DISCUSSION TITLE
Alist丢失有没有什么找回的方法

### DISCUSSION THREAD

**Discussion by Hewgy** [AlistGo/alist#3352] [Q&A] [answered]
我之前挂载Alist的电脑坏了，然后alist的本地文件全丢了，重装系统之后还能找回原本挂载的网盘吗

**Comment by anwen-anyi** [comment, marked as answer]
只要你还有数据库文件就能 数据库文件没了就找不回了
数据库文件在 Alist 同级目录下 data/data.db

**Reply by Hewgy** [reply]
难顶，我这个应该是找不回来了，我备份都还没做就坏了，文件一起跟着丢了，感谢回答
